h2o-core/src/main/java/hex/Model.java

```diff
@@ -60,7 +61,7 @@
     final Key[] modelKeys = KeySnapshot.globalSnapshot().filter(new KeySnapshot.KVFilter() {
       @Override
       public boolean filter(KeySnapshot.KeyInfo k) {
-        return Value.isSubclassOf(k._type, Model.class);
+        return Value.isSubclassOf(k._type, Model.class) && !Value.isSubclassOf(k._type, QuantileModel.class);
```

Comment:
Or we could hide models that are supposed to be invisible to our users under user-hidden keys (`Key.makeUserHidden`) and only filter out those here.

It'd require a slight modification to ModelBuilder.

Reply:
Yup, that is a good idea. I started with the least intrusive change as the `list` method would fail anyway when the quantiles model exists and I wasn't sure if any clients are accessing the quantiles model via some other paths.

Reply:
I'm ok with it.